tests/test_jaxpr_translator_driver.py

```diff
@@ -0,0 +1,360 @@
+# JaCe - JAX Just-In-Time compilation using DaCe (Data Centric Parallel Programming)
+#
+# Copyright (c) 2024, ETH Zurich
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Implements some tests of the subtranslator driver."""
+
+from __future__ import annotations
+
+import re
+
+import dace
+import pytest
+from dace.data import Array, Data, Scalar
+
+from jace import translator as jtrans
+from jace.util import JaCeVar
+
+
+@pytest.fixture(scope="module")
+def alloc_driver():
```

Comment:
Fixtures shouldn't be named as verbs but as substantives, since they are used as proxies for their result.
  
```suggestion
def translation_driver():
```